src/AssetMapper/SassPublicPathAssetPathResolver.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Symfonycasts\SassBundle\AssetMapper;
+
+use Symfony\Component\AssetMapper\Path\PublicAssetsPathResolverInterface;
+
+class SassPublicPathAssetPathResolver implements PublicAssetsPathResolverInterface
+{
+    public function __construct(private readonly PublicAssetsPathResolverInterface $decorator)
+    {}
+
+    public function resolvePublicPath(string $logicalPath): string
+    {
+        $path = $this->decorator->resolvePublicPath($logicalPath);
+
+        if (str_contains($path, '.scss')) {
```

Comment:
Ideally we would target JUST the one `.scss` file. Since we are only given the logical path here... and the `root_sass` config is an absolute path, that makes it tricky.

WDYT about changing the `root_sass` to be the asset mapper "logical path" instead? Then it would be easy to compare that logical path to this logical path. And, it makes sense: your "input Sass" file MUST be in an AssetMapper path for all of this to work. If we decide to make that change and it works, I'll do the same for tailwind-bundle.

Reply:
Hum sorry I don't get what you mean here! Maybe something I miss understand with asset mapper